src/terms/kinetic.jl

```diff
@@ -9,7 +9,7 @@ struct TermKinetic <: Term
     kinetic_energies::Vector{Vector} # kinetic energy 1/2|G+k|^2 for every kpoint
 end
 function TermKinetic(basis::PlaneWaveBasis)
-    kinetic_energies = [[sum(abs2, basis.model.recip_lattice * (G+kpt.coordinate))/2
+    kinetic_energies = [[sum(abs2, basis.model.recip_lattice * (G + kpt.coordinate)) / 2
```

Comment:
I like the no-space for the + here, but that's just taste

Reply:
I find it easier to parse the G with the space, actually.